node/resolvers/Queries/Settings.ts

```diff
@@ -78,18 +116,33 @@ const B2BSettings = {
   },
   getAccount: async (_: void, __: void, ctx: Context) => {
     const {
-      clients: { lm },
+      clients: { lm, audit },
+      ip
     } = ctx
 
-    return lm.getAccount().catch((e) => {
+    try {
+      
+      await audit.sendEvent({
+        subjectId: 'get-account-event',
+        operation: 'GET_ACCOUNT',
+        meta: {
+          entityName: 'Account',
+          remoteIpAddress: ip,
+          entityBeforeAction: JSON.stringify({}),
+          entityAfterAction: JSON.stringify({}),
+        },
+      })
+
+      return await lm.getAccount()
```

Comment:
this way we won't emit the event if the get failed.
```suggestion
      const result = await lm.getAccount()
      await audit.sendEvent({
        subjectId: 'get-account-event',
        operation: 'GET_ACCOUNT',
        meta: {
          entityName: 'Account',
          remoteIpAddress: ip,
          entityBeforeAction: JSON.stringify({}),
          entityAfterAction: JSON.stringify({}),
        },
      })

      return result
```